Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -724,6 +724,7 @@ class BrowserViewController: UIViewController {
 
         // Cliqz: bring footer to front so that the tab toolbar will be visible all time
         self.view.bringSubview(toFront: footer)
+        self.bringVpnToFrontIfShown()
```

Comment:
When keyboard shown  line 726 is bringing toolbar in front which results to have toolbar in front of VPN controller. This code will make sure to have vpn controller always on top of toolbar.